ckan/tests/logic/action/test_get.py

```diff
@@ -768,6 +768,12 @@ def test_user_list_default_values(self):
         assert 'email' not in got_user
         assert 'datasets' not in got_user
 
+    @helpers.change_config('ckan.auth.allow_anonymous_access', 'false')
+    def test_user_list_raises_not_authorized_if_config_allow_anon_access_false(self):
+        user = factories.User()
+        nose.tools.assert_raises(logic.NotAuthorized, helpers.call_action,
```

Comment:
user_list will also depend on the ckan.auth.public_user_details = False config. It defaults to true so this would have to be set first as well.